tez-dag/src/main/java/org/apache/tez/dag/app/rm/DagAwareYarnTaskScheduler.java

```diff
@@ -743,7 +743,7 @@ public float getProgress() {
 
       ++numHeartbeats;
       if (LOG.isDebugEnabled() || numHeartbeats % 50 == 1) {
-        LOG.info(constructPeriodicLog(freeResources));
+        LOG.debug(constructPeriodicLog(freeResources));
```

Comment:
This is a new change from the previous patch and a good catch that has been there since this file was created.